--- a/scripts/install-debian.sh
+++ b/scripts/install-debian.sh
@@ -1,5 +1,5 @@
 #!/bin/bash
-# This script installs Klipper on an debian
+# This script install Klipper on an YoctianOS 
 #
 
 PYTHONDIR="${HOME}/klippy-env"
@@ -10,17 +10,17 @@
 # Step 1: Install system packages
 install_packages()
 {
-    # Packages for python cffi
-    PKGLIST="virtualenv python-dev libffi-dev build-essential"
+    # Packages for python3 cffi 
+    PKGLIST="python3-venv python3-dev libffi-dev packagegroup-core-buildessential" 
     # kconfig requirements
-    PKGLIST="${PKGLIST} libncurses-dev"
+    PKGLIST="${PKGLIST} ncurses-dev" 
     # hub-ctrl
-    PKGLIST="${PKGLIST} libusb-dev"
+    PKGLIST="${PKGLIST} libusb-1.0-dev" 
     # AVR chip installation and building
-    PKGLIST="${PKGLIST} avrdude gcc-avr binutils-avr avr-libc"
+    PKGLIST="${PKGLIST} avrdude avr-gcc avr-binutils avr-libc" 
     # ARM chip installation and building
-    PKGLIST="${PKGLIST} stm32flash libnewlib-arm-none-eabi"
-    PKGLIST="${PKGLIST} gcc-arm-none-eabi binutils-arm-none-eabi libusb-1.0 pkg-config"
+    PKGLIST="${PKGLIST} stm32flash" 
+    PKGLIST="${PKGLIST} yoctianos-arm-none-eabi libusb-1.0-0 pkgconfig" 
 
     # Update system package info
     report_status "Running apt-get update..."
@@ -37,7 +37,7 @@
     report_status "Updating python virtual environment..."
 
     # Create virtualenv if it doesn't already exist
-    [ ! -d ${PYTHONDIR} ] && virtualenv -p python2 ${PYTHONDIR}
+    [ ! -d ${PYTHONDIR} ] && python3 -m venv ${PYTHONDIR} 
 
     # Install/update dependencies
     ${PYTHONDIR}/bin/pip install -r ${SRCDIR}/scripts/klippy-requirements.txt
